packages/loader/container-loader/src/test/deltaManager.spec.ts

```diff
@@ -332,6 +332,12 @@ describe("Loader", () => {
                     deltaManager.connectionManager.forceReadonly(true);
                 });
             });
+
```

Comment:
You could test it at that level (at least the check in DeltaManager, so yes, adding some code to do so would be nice).